controllers/controller.go

```diff
@@ -103,7 +103,6 @@ func (s *GetListDockerControllerImpl) GetDockerConfigList() http.HandlerFunc {
 // DockerForm  display DockerForm page
 func (s *DockerListFormImpl) DockerForm() http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
-		home := os.Getenv("HOME")
 		tmpl, err := template.ParseFiles(home + "/view/DockerForm.html")
 		if err != nil {
 			fmt.Println(errors.New("unable to execute the template"))
```

Comment:
instaed of giving an error message "unable to execute the template" ,you can return http message with an error.

Reply:
fixed